Assets/MRTK/SDK/Features/Utilities/Solvers/HandConstraintPalmUp.cs

```diff
@@ -90,6 +93,46 @@ public float FollowHandCameraFacingThresholdAngle
             set => followHandCameraFacingThresholdAngle = value;
         }
 
+        [SerializeField]
+        [Tooltip("With this active, solver will activate the UI after the palm threshold has been met and the user gazes at the activation point")]
+        private bool useGazeActivation = false;
+
+        /// <summary> 
+        /// With this active, solver will activate after the palm threshold has been met and the user gazes at the activation point
+        /// </summary>
+        public bool UseGazeActivation
+        {
+            get => useGazeActivation;
+            set => useGazeActivation = value;
+        }
+
+        [SerializeField]
+        [Tooltip("The distance between the planar intersection of the eye gaze ray and the activation transform. Uses square magnitude between two points for distance")]
+        [Range(0.0f, .1f)]
+        private float gazeProximityThreshold = .005f;
+
+        /// <summary>
+        /// The distance between the planar intersection of the eye gaze ray and the activation transform
+        /// </summary>
+        public float GazeProximityThreshold
+        {
+            get => gazeProximityThreshold;
+            set => gazeProximityThreshold = value;
+        }
+
+        private bool targetWorldLocked = false;
+
+        /// <summary>
+        /// Is the current solver object world-locked? If so, don't compute whether it's a valid controller.
```

Comment:
>If so, don't compute whether it's a valid controller.

Recommend removing this last part since it is highly specific to this class, and this comment will be visible in the public API docs. Not sure a public consumer would understand, it might confuse them instead.

Reply:
Can do!